src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+body,
+.nav {
   margin: 0;
+  padding: 0;
+  max-width: 1200px;
```

Comment:
The max-width property should be applied to the .header class instead of body and .nav to make sure the layout fits on 1024px and 1200px.

Reply:
![man-mechanic_1f468-200d-1f527](https://github.com/mate-academy/layout_moyo-header/assets/129754902/f4efc3e6-9244-4efd-a690-ca196e7f68dd)